options/locale/locale_en-US.ini

```diff
@@ -2310,6 +2311,7 @@ create_org = Create Organization
 repo_updated = Updated
 people = People
 teams = Teams
+code = Code
```

Comment:
Could these strings ever be different?

I guess would a language need to translate this as <User Code> <Organization Code>?

I guess the other thing is I think the translation key here might need to be source_code rather than just code

Reply:
they might be or not but again I did not want to refactor out of scope code to not make this PR too large. And translation key `code` was left to be consistent with other places